Errors on draft up on nodejs example

This was the second attempt. I had a successful first attempt but nuked the scaffolding files and wanted to try again. Running under github.com/Azure/draft/examples/nodejs

```
$ draft up
--> Building Dockerfile
Step 1 : FROM node:onbuild
# Executing 5 build triggers...
Step 1 : ARG NODE_ENV
 ---> Using cache
Step 1 : ENV NODE_ENV $NODE_ENV
 ---> Using cache
Step 1 : COPY package.json /usr/src/app/
 ---> Using cache
Step 1 : RUN npm install && npm cache clean
 ---> Running in 49b52780240f
npm info it worked if it ends with ok
npm info using npm@5.0.0
npm info using node@v8.0.0
npm info lifecycle example-nodejs@0.0.0~preinstall: example-nodejs@0.0.0
npm info linkStuff example-nodejs@0.0.0
npm info lifecycle example-nodejs@0.0.0~install: example-nodejs@0.0.0
npm info lifecycle example-nodejs@0.0.0~postinstall: example-nodejs@0.0.0
npm info lifecycle example-nodejs@0.0.0~prepublish: example-nodejs@0.0.0
npm info lifecycle example-nodejs@0.0.0~prepare: example-nodejs@0.0.0
npm info lifecycle undefined~preshrinkwrap: undefined
npm info lifecycle undefined~shrinkwrap: undefined
npm notice created a lockfile as package-lock.json. You should commit this file.
npm info lifecycle undefined~postshrinkwrap: undefined
npm WARN example-nodejs@0.0.0 No description
npm WARN example-nodejs@0.0.0 No repository field.
npm WARN example-nodejs@0.0.0 No license field.

up to date in 0.15s
npm info ok
npm info it worked if it ends with ok
npm info using npm@5.0.0
npm info using node@v8.0.0
npm ERR! As of npm@5, the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid. If you want to make sure everything is consistent, use 'npm cache verify' instead.
npm ERR!
npm ERR! If you're sure you want to delete the entire cache, rerun this command with --force.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-05-31T22_45_42_521Z-debug.log
Error encountered streaming JSON response: The command '/bin/sh -c npm install && npm cache clean' returned a non-zero code: 1
Watching local files for changes...
```

--- a/pkg/draft/defaultpacks/node.go
+++ b/pkg/draft/defaultpacks/node.go
@@ -42,7 +42,7 @@ fi
 echo Node.js
 `
 
-const nodeDockerfile = `FROM node:onbuild
+const nodeDockerfile = `FROM node:6-onbuild
 EXPOSE 8080
 RUN npm install
 CMD ["npm", "start"]

